packages/utils/src/saml/saml-server.ts

```diff
@@ -0,0 +1,101 @@
+// This is a helper script that lets you test constructing your own soap+saml requests. It creates the SOAP Envelope and then sends it to to the gateway specified in the request body.
+// npm run saml-server and then reference the Metriport- IHE GW / XML + SAML Constructor - Postman collection
+
+import express from "express";
+import { json, Request, Response } from "express";
+import axios from "axios";
+import fs from "fs";
+import https from "https";
+
+import { createAndSignXCPDRequest } from "@metriport/core/external/saml/xcpd/iti55-envelope";
+import { createAndSignDQRequest } from "@metriport/core/external/saml/xca/iti38-envelope";
+import { createAndSignDRRequest } from "@metriport/core/external/saml/xca/iti39-envelope";
+import { getEnvVarOrFail } from "@metriport/core/util/env-var";
+
+import * as dotenv from "dotenv";
```

Comment:
nit: keep imports at the top with no empty lines in between them so we can use VSCode's organize import and not have unimportant import changes between commits/PRs, please